ocaml/testsuite/tests/jane-modular-syntax/user_error3_1.compilers.reference

```diff
@@ -0,0 +1,4 @@
+File "user_error3_1.ml", line 9, characters 31-64:
+9 | let _misnamed_extension = () [@jane.this_extension_doesn't_exist];;
+                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Error: Cannot have an attribute named [@jane.this_extension_doesn't_exist]: Missing a feature component
```

Comment:
This test should now be for `jane.erasable.this_extension_doesn't_exist`

Reply:
I think you solved this problem through the dual renaming (the no-such-extension one getting an extra dot), which is just as good.